admin-dev/themes/new-theme/js/components/navbar-handler.ts

```diff
@@ -70,13 +70,8 @@ export default class NavbarHandler {
   }
 
   updateBrowserHash(target: string): void {
-    const hashName = target.replace('#', `#${this.tabPrefix}`);
-
-    if (window.history.pushState) {
```

Comment:
ping @NeOMakinG can you have a look please

Reply:
@jolelievre did you test this? And using the previous and next func of the browser? We had an issue on this on FO where using window.location is erasing the history, but as it's only aiming the hash part, maybe not on this case

Reply:
This was only used to change the hashtag indeed, but I don't think history can handle this anyway 🤔

Even if it could losing history between the page's tabs is not that much a problem IMHO, compared to the previous issue, when the form shop context was saved you were always redirected to the first tab Which is really annoying when you want to compare the prices between the shops (for example)

I wanted to keep the `pushState` but there is no event related to it, unlike when using `window.location.hash` where a `hashchange` event is triggered. So it allows keeping track on the current hash and to update the URLs in the multishop header.

I tried to find something similar for `pushState` but since no event exists the only workaround I found actually overrides the function in the window object to integrate a custom event when the method is called. Which seemed overkill to me, and not really a clean solution.